### ISSUE TITLE
No longer possible to create new Menu items in Manager Top Menu

### ISSUE THREAD

**Issue by BobRay** [modxcms/revolution#15681]
### Summary
New menu items with the parent "Top Navigation" no longer appear. Top menu items created previously no longer appear in the menu.

### Step to reproduce
Go to System -> Menus.
Click on "Create Menu"
Create a new menu item called MyFiles with the parent "Top Navigation" with or without namespace or permission and save it.

### Observed behavior

In the menu tree, the item appears as >_ MyFiles (unlike the other menu items)
It does not appear in the menu.

I've had Custom Search in my main menu for a long time. It still shows in the tree as >_Custom Search, but does not appear in the menu.

### Expected behavior
Menu item should appear in the Manager's Main Menu.

### Environment
MODX 2.8.1

**Comment by ghost** [comment]
It looks like it needs an action and a namespace. Without either of those two, no menu. With them, even if they are invalid, the menu item shows. All menu items without children look like that in the menu tree, they always have.

<img width="400" alt="Screen Shot 2021-04-21 at 8 24 33" src="https://user-images.githubusercontent.com/1073780/115501365-c386ca80-a27b-11eb-93c6-9c2172f8435f.png">

**Comment by BobRay** [comment]
Good catch. Action used to be a drop-down with (none) as an option.  I see that typing (none) still works and the menu item shows if you put (none) for the action and add a namespace.

If both an action and a namespace are required, the menu shouldn't save without them.

**Comment by smg6511** [comment]
In both 2.8.x and 3.0.0-alpha only the Action is necessary to make the menu item show in the interface. However, to make the menu item work both a valid Namespace and Action are needed. I think both fields should be changed to be required and the validation message can spell out that only placeholder values are needed for these fields initially.
